src/types.ts

```diff
@@ -295,13 +296,14 @@ export interface TransactionResponse {
     serviceFee?: number;
     feeCurrency: string;
     amlScreeningResult?: AmlScreeningResult;
+    operation: TransactionOperation;
     customerRefId?: string;
     amountInfo?: AmountInfo;
     feeInfo?: FeeInfo;
-    signedMessages?: SignedMessageResponse[];
+    signedMessages: SignedMessageResponse[];
     extraParameters?: any;
     externalTxId?: string;
-    destinations?: TransactionResponseDestination[];
+    destinations: TransactionResponseDestination[];
```

Comment:
`destination` is an optional field

Reply:
An array is always returned, so it is always defined.
It is likely done this way because that makes the end-user work easier, you can just look through it whether it is empty or not.